Allure.NUnit.AsyncTests/AsyncLifeCycleTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Threading.Tasks;
+using Allure.Net.Commons;
+using NUnit.Allure.Attributes;
+using NUnit.Allure.Core;
+using NUnit.Framework;
+
+namespace Allure.NUnit.AsyncTests
+{
+    [TestFixture]
+    [AllureNUnit(wrapIntoStep: false)]
+    internal class AsyncLifeCycleTests
+    {
+        [Test]
+        public async Task AsyncStepWithWrapInStep()
+        {
+            Console.WriteLine("AsyncStepWithWrapInStep: calling AsyncActionWithInnerStep ...");
+            await AllureLifecycle.Instance.WrapInStepAsync(
```

Comment:
That's good you provide an examples for internal methods.
But anyway - end user should not call AllureLifecycle methods at all.
I suggest to write separate unit-tests for this.

Reply:
Note: yes, some examples still contains AllureLifecycle method calls, but it will be deprecated in major updates

Reply:
I see that some users actively use `AllureLifecycle.Instance.WrapInStep` in their projects - so I've added async version of these methods and provided unit tests on this methods. :) Maybe you will set `[Obsolette]` on them lately in next major release? :)